### ISSUE TITLE
Is it possible to download query results?

### ISSUE THREAD

**Issue by agricolamz** [macleginn/eurasian-phonologies#1]
It will be so nice to have a download button at the end of all results pages such as [this](http://eurasianphonology.info/search_feature?query=glottal)...

**Comment by macleginn** [comment]
Yes, it would. There was a file-export version before I started rendering HTML on the server. I will add it back at some point. (BTW, these results you can actually copy.)

**Comment by agricolamz** [comment]
Yes, I know, but I want to add an ability to download the data via lingtypology (check out new release news), so it will be easier to make automatic API.
